pages/1.10/installing/production/deploying-dcos/installation/index.md

```diff
@@ -156,18 +156,21 @@ By default, DC/OS clusters have [fault domain awareness](/1.12/deploying-service
 
 1. Create a fault domain detect script named `fault-domain-detect` to run on each node to detect the node's fault domain. During installation, the output of this script is passed to Mesos.
 
-   We recommend the format for the script output be:
-
-```json
-  {
-    "fault_domain": {
-      "region": {
-        "name": <region>,
-        "zone": <zone>
-      }
+   The format for the script output is:
```

Comment:
I would suggest that we use a 'tab length' of 2 spaces rather than 4 here so that we're more in line with the rest of our documentation formatting. (this applies to each instance of this JSON block within this PR)

Reply:
4 spaces is what currently works with our system and code blocks, I could put it to two within the json code, but the 4 in front of the backtics is necessary. Our markdown system is not the github one.

Reply:
Thanks @ck4adventure !